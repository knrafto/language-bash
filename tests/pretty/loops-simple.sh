for x in true; do true; done
until true; do true; done
while true; do true; done
