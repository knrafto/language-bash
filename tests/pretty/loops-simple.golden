for x in true; do
    true;
done;
while true; do
    true;
done;
